lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -6621,6 +6658,18 @@ namespace Js
         return prototype;
     }
 
+    DynamicObject* JavascriptLibrary::CreateClassConstructorPrototypeObject(RecyclableObject * protoParent)
```

Comment:
(nit) What would you think about naming this `CreateClassPrototypeObject` (and similarly removing "Constructor"/"Ctor" from other new names)? It's inherently confusing whether we are talking about the class prototype property or the constructor's proto parent, but for me "class prototype" is more natural for describing the former.

Reply:
That's fine with me.